src/spdx2/agent/spdx2.php

```diff
@@ -198,24 +198,24 @@ protected function getUri($fileBase,$packageName)
    * @param string[] $licenses
    * @return string
    */
-  protected function implodeLicenses($licenses)
+  protected function implodeLicenses($licenses, $prefix="")
   {
     sort($licenses);
     if(count($licenses) == 3 &&
        ($index = array_search("Dual-license",$licenses)) !== false)
     {
-      return $licenses[$index===0?1:0] . " or " . $licenses[$index===1?2:1];
+      return $prefix . $licenses[$index===0?1:0] . " OR " . $prefix . $licenses[$index===1?2:1];
```

Comment:
In the case that the licenses are "BSD", "GPL" and "Dual-License" the returned string is
```
"BSD OR GPL"
```
without parentheses.

In the other case that the licenses are "Zlib" and "BSD OR GPL" the returned string will be
```
"Zlib AND (BSD OR GPL)"
```
with the added parantheses from line R212

That are the two supported cases.

(I will add unit tests)